cmake/modules/hunter_copy_license_file.cmake

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2015, Aaditya Kalsi
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+function(hunter_copy_license_file srcdir dstfile)
+  # find the license file to copy
+  
+  if(EXISTS "${srcdir}/LICENSE")
+    file(COPY "${srcdir}/LICENSE" ${dstfile})
+    return()
+  endif()
+
+  if(EXISTS "${srcdir}/LICENSE.txt")
+    file(COPY "${srcdir}/LICENSE.txt" ${dstfile})
+    return()
+  endif()
+
+  if(EXISTS "${srcdir}/COPYING")
+    file(COPY "${srcdir}/COPYING" ${dstfile})
+    return()
+  endif()
+
+  if(EXISTS "${srcdir}/COPYING.txt")
+    file(COPY "${srcdir}/COPYING.txt" ${dstfile})
+    return()
+  endif()
+
+  if(EXISTS "${srcdir}/license")
+    file(COPY "${srcdir}/license" ${dstfile})
+    return()
+  endif()
+
+  if(EXISTS "${srcdir}/license.txt")
+    file(COPY "${srcdir}/license.txt" ${dstfile})
+    return()
+  endif()
+
+  if(EXISTS "${srcdir}/copying")
+    file(COPY "${srcdir}/copying" ${dstfile})
+    return()
+  endif()
+
+  if(EXISTS "${srcdir}/copying.txt")
+    file(COPY "${srcdir}/copying.txt" ${dstfile})
+    return()
+  endif()
+
+  if(EXISTS "${srcdir}/LICENSE_1_0.txt")
```

Comment:
In case we haven't find anything we can use `file(GLOB "LICENSE_*.txt" ...)` here + check that only one file found should be enough.